tests/sentry/integrations/slack/service/test_slack_service.py

```diff
@@ -499,3 +624,40 @@ def test_send_notification_to_slack_channel_error(self):
                 client=SlackSdkClient(integration_id=self.integration.id),
             )
         assert err.value.response["ok"] is False
+
+    def test_get_channel_id_from_parent_notification_notification_action(self):
+        channel_id = self.service._get_channel_id_from_parent_notification_notification_action(
+            parent_notification=self.parent_notification_action,
+        )
+        assert channel_id == "C0123456789"
```

Comment:
It may be good to hardcode this somewhere as a `TEST_CHANNEL_ID` variable or something, just to make it clear this is something this test is configuring somewhere.